### PR TITLE
Fix outdated references to `core` namespace in deeplink documentation

### PR DESCRIPTION
**Pull request by AE-MS**
The documentation for `executedDeepLink` referenced an outdated/removed namespace (`core`) and was fixed.

I found one other outdated reference to `core` in the test app and also fixed it.

### CODE REVIEW

File: apps/teams-test-app/src/components/PagesAPIs.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ const NavigateToApp = (): React.ReactElement =>
 
 const ShareDeepLink = (): ReactElement =>
   ApiWithTextInput<DeepLinkParameters & ShareDeepLinkParameters>({
-    name: 'core.shareDeepLink',
+    name: 'pages.shareDeepLink',
```
Comment (erinha) [inline apps/teams-test-app/src/components/PagesAPIs.tsx:88]: I'm curious if the e2e tests will fail with this change (unless you already updated them in the corresponding host sdk repo).

File: change/@microsoft-teams-js-59318f7e-dc90-4d48-a1b2-5616fb3a6cef.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Fixed outdated references to 'core.' namespace in documentation",
```
Comment (erinha) [inline change/@microsoft-teams-js-59318f7e-dc90-4d48-a1b2-5616fb3a6cef.json:3]: Preference is to enclose namespaces or other API types/interfaces/etc in backtick quotes.

**Review by erinha — CHANGES_REQUESTED** [review]
Thank you for making this change! Left a question about e2e test with this update and comment about formatting of change file.

**Review by erinha — APPROVED** [review]
🚀

**Review by Ella-ly — APPROVED** [review]

**Review by TrevorJoelHarris — APPROVED** [review]

**Review by jadahiya-MSFT — APPROVED** [review]

**Pull request merged**